### PR TITLE
[with-tailwind] fix FOUC + improve Next.js app

### PR DESCRIPTION
**Pull request by axeldelafosse**
# Why?

Closes https://github.com/nandorojo/solito/issues/144.

Also, you forgot to wrap the `View`. You should use `const StyledView = styled(View)` or create a wrapper file in your design system `app/design`.

Bonus points for using [`@showtime-xyz/universal.view`](https://github.com/showtime-xyz/showtime-frontend/blob/staging/packages/design-system/view/index.tsx) to avoid this kind of mistakes!

It was working because we use the Babel plugin but we can now get rid of Babel and start to use SWC with Next.js!

# How?

- Moved the React Native for Web reset to the global CSS file
- Removed the Babel config
- Used the Reanimated SWC plugin

# Test

Run the Tailwind example web app

### CODE REVIEW

File: example-monorepos/with-tailwind/apps/next/global.css [hunk 1]
```diff
@@ -1,2 +1,43 @@
+@tailwind base;
 @tailwind components;
 @tailwind utilities;
+
```
Comment (nandorojo) [inline example-monorepos/with-tailwind/apps/next/global.css:4]: Can we move this CSS back into `_document` directly? We should discourage `.css` files.

File: example-monorepos/with-tailwind/apps/next/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "@expo/next-adapter": "^4.0.12",
     "app": "*",
-    "next": "^12.3.0",
```
Comment (nandorojo) [inline example-monorepos/with-tailwind/apps/next/package.json:14]: Why the downgrade?

File: example-monorepos/with-tailwind/packages/app/features/user/detail-screen.tsx [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
 const { useParam } = createParam<{ id: string }>()
 
 export function UserDetailScreen() {
   const [id] = useParam('id')
 
   return (
-    <View className="flex-1 justify-center items-center">
-      <Text className="text-center font-bold mb-4">{`User ID: ${id}`}</Text>
+    <View className="flex-1 items-center justify-center">
```
Comment (nandorojo) [inline example-monorepos/with-tailwind/packages/app/features/user/detail-screen.tsx:12]: Good catch!

**Review by nandorojo — CHANGES_REQUESTED** [review]
Thanks for this @axeldelafosse! Left a few small notes.

File: example-monorepos/with-tailwind/apps/next/package.json (hunk 2 above)
Comment (axeldelafosse) [reply example-monorepos/with-tailwind/apps/next/package.json:14]: Yeah the SWC version doesn't match

File: example-monorepos/with-tailwind/apps/next/global.css (hunk 1 above)
Comment (axeldelafosse) [reply example-monorepos/with-tailwind/apps/next/global.css:4]: Yeah I'm not sure to understand why it's not working as it used to... Using the `global.css` file for this specific use-case seems OK in my opinion. Adding the comment!

**Pull request merged**